dotnet/aspnet/mssqldbWithTests/Application/SampleWebApplication.FunctionalTests/SampleFunctionalTests.cs

```diff
@@ -47,12 +46,14 @@ public void SampleFunctionalTest1()
                     Assert.AreEqual("Home Page - My ASP.NET Application", driver.Title, "Expected title to be 'Home Page - My ASP.NET Application'");
                     break;
                 }
-                catch
+                catch(Exception e)
                 {
                     var currentTimestamp = DateTime.Now.Millisecond;
                     if(currentTimestamp > endTimestamp)
                     {
-                        throw;
+                        Console.Write("##vso[task.logissue type=error;]Test SampleFunctionalTest1 failed with error: " + e.ToString());
```

Comment:
let not have task specific constructs in the code

Reply:
Without this we won't get logs in Kusto.. Whatever we write to stderr goes to test results directly and skips logging to console, so this is how we can print to both console and test results